.github/workflows/ci.yml

```diff
@@ -158,6 +158,42 @@ jobs:
         with:
           name: "${{ matrix.IMAGE.TOXENV }} on ${{ matrix.IMAGE.IMAGE }}"
 
+  linux-arm64:
+    runs-on: [self-hosted, linux, ARM64]
+    container: ghcr.io/pyca/cryptography-runner-${{ matrix.IMAGE.IMAGE }}
+    strategy:
+      fail-fast: false
+      matrix:
+        IMAGE:
+          - {IMAGE: "ubuntu-focal:aarch64", TOXENV: "py38"}
+    name: "${{ matrix.IMAGE.TOXENV }} on ${{ matrix.IMAGE.IMAGE }}"
+    timeout-minutes: 20
+    steps:
+      - name: "Delete workspace" # self-hosted runners need this, sigh
+        run: find ! -name '.' ! -name '..' -delete
```

Comment:
Are there any docs on the need for this?

Reply:
There's a variety of support threads and actions people have built for this, but https://github.community/t/how-to-properly-clean-up-self-hosted-runners/128909 is an example. GitHub's docs don't bother mentioning it, although they say self-hosted runners "Don't need to have a clean instance for every job execution." (https://docs.github.com/en/actions/hosting-your-own-runners/about-self-hosted-runners)